### PR TITLE
Adding openmp image

### PR DESCRIPTION
**Pull request by SwRaw**
https://advanced-micro-devices-demo--2323.com.readthedocs.build/en/2323/reference/openmp/openmp.html#openmp-toolchain

### CODE REVIEW

**Review by samjwu — DISMISSED** [review]

**Comment by samjwu** [comment]
https://advanced-micro-devices-demo--2323.com.readthedocs.build/en/2323/reference/openmp/openmp.html

**Comment by keryell** [comment]
Would it be possible to have `git` friendly formats instead of binaries.
A vector format would be event better.
What about SVG?

**Comment by Rmalavally** [comment]
@keryell Thank you for your feedback. Let me check and get back to you.

**Review by Rmalavally — APPROVED** [review]

File: docs/data/reference/openmp/openmp_toolchain.jpg
Comment (saadrahim) [inline docs/data/reference/openmp/openmp_toolchain.jpg:1]: The jpg file should likely be deleted if we can use just the svg.

**Review by saadrahim — CHANGES_REQUESTED** [review]

File: docs/data/reference/openmp/openmp_toolchain.jpg
Comment (Rmalavally) [reply docs/data/reference/openmp/openmp_toolchain.jpg:1]: That's right. 

@SwRaw Could you please delete the jpg file? Thanks so much.

Comment (samjwu) [reply docs/data/reference/openmp/openmp_toolchain.jpg:1]: Done in [5321e2e](https://github.com/RadeonOpenCompute/ROCm/pull/2323/commits/5321e2ee3e463cd55eec599dc7f689d317f55ffc)

**Review by saadrahim — APPROVED** [review]

**Pull request merged**